src/sentry/api/endpoints/organization_trace_item_attributes_ranked.py

```diff
@@ -54,8 +57,62 @@ def get(self, request: Request, organization: Organization) -> Response:
             sampling_mode=snuba_params.sampling_mode,
         )
 
-        query_1 = request.GET.get("query_1", "")
-        query_2 = request.GET.get("query_2", "")
+        function_string = request.GET.get("function", "count(span.duration)")
+        above = request.GET.get("above") == "1"
+
+        match = fields.is_function(function_string)
+        if match is None:
+            raise InvalidSearchQuery(f"{function_string} is not a function")
+
+        function_name = match.group("function")
+        columns = match.group("columns")
+        arguments = fields.parse_arguments(function_name, columns)
+
+        if len(arguments) != 1:
+            raise InvalidSearchQuery(
+                f"Function {function_name} must have exactly one argument, got {len(arguments)}"
+            )
```

Comment:
if args != 1, it just means the `if should_segment_suspect_cohort` code block shouldn't run - we should still be able to run the analysis just based on the time bounds?